fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/expressions/RelationalExpression.java

```diff
@@ -680,7 +681,7 @@ default AliasMap bindIdentities(@Nonnull final RelationalExpression otherExpress
      */
     @Nonnull
     default Iterable<MatchInfo> subsumedBy(@Nonnull final RelationalExpression candidateExpression,
-                                           @Nonnull final AliasMap aliasMap,
+                                           @Nonnull final AliasMap bindingAliasMap,
```

Comment:
I thought we were going to add an optional of the `translationMap` that is constructed of the composition of the max match maps from below, or directly if leaves.

Reply:
Yes that was the plan but I couldn't do it because of the ambiguity bug, so, not in this PR